server/web/server.go

```diff
@@ -76,6 +77,12 @@ func NewHttpServerWithCfg(cfg *Config) *HttpServer {
 // MiddleWare function for http.Handler
 type MiddleWare func(http.Handler) http.Handler
 
```

Comment:
Why there is no any implement of LifeCycleCallback ?

Reply:
`LifeCycleCallback` will be used for other extension or plugin, to custom logic into beego server lifecycle.